security/self-assessment.md

```diff
@@ -0,0 +1,453 @@
+  security relevant component. The Functions [design
+  doc](https://github.com/crossplane/crossplane/blob/main/design/design-doc-composition-functions.md)
+  and the Functions
+  [specification](https://github.com/crossplane/crossplane/blob/main/contributing/specifications/functions.md)
+  are useful related resources.
+* Webhooks: Crossplane has a set of validating admission webhooks enabled by
+  default that perform validation on user provided resources in the composition
+  model, such as `XRDs` and `Compositions`. The provided objects are validated
+  according to their schema and invalid objects will be rejected from the API
+  server. These webhooks are optional and can be disabled.
+
+## Project Compliance
+
+The Crossplane project is not documented to comply with any specific security
+standards or sub-sections at this time.
+
+## Secure Development Practices
+
+### Development Pipeline
+
+* All PRs on the Crossplane project require at least one approval from a code
+  owner or maintainer
+* A DCO sign-off is required for all commits, but cryptographically signed
+  commits are not currently required
+* A CI/CD pipeline with extensive checks runs on all PRs, `main` branch, and
+  release branches. The checks performed by this
+  [pipeline](https://github.com/crossplane/crossplane/blob/main/.github/workflows/ci.yml)
+  include:
+  * A E2E test suite to validate all major scenarios and features
+  * A unit test suite to validate all unit level functionality
+  * `golangci-lint` runs with all linters turned on by default (`enable-all:
+    true`), and a handful disabled with justifications provided. The
+    [`gosec`](https://github.com/securego/gosec) linter is included to inspect
+    the source code for security problems.
+  * A CodeQL scan performs static analysis to identify potential vulnerabilities
+  * The Trivy vulnerability scanner is run over the codebase in `fs` mode
+  * Fuzz testing is performed on the codebase with multiple test cases developed
+    during the [Fuzzing
+    audit](https://github.com/crossplane/crossplane/blob/main/security/ADA-fuzzing-audit-22.pdf)
+* The `crossplane/crossplane` image is based on the `distroless/static` base
+  image to minimize surface area of the container image
+
+### Communication Channels
+
+* Internal: Most communication amongst the team occurs in the [Crossplane
+  Slack](https://slack.crossplane.io/) dedicated workspace (in particular the
+  `#core-development` channel), as well as the Github
+  [`crossplane`](https://github.com/crossplane) and
+  [`crossplane-contrib`](https://github.com/crossplane-contrib) organizations on
+  issues, PRs, and discussions.
+* Inbound: Communication from users of Crossplane most commonly occurs on the
+  Crossplane Slack `#general` channel, in regular Community meetings, and other
+  channels outlined in the main
+  [README](https://github.com/crossplane/crossplane?tab=readme-ov-file#get-involved).
```

Comment:
maybe worth mentioning https://github.com/crossplane/crossplane/security/policy#reporting-a-vulnerability too somewhere here?

Reply:
I now saw it a few lines down 😅